README.md

```diff
@@ -102,24 +103,24 @@ Where you replace <USER_NAME> by your username in the target organization.
 ### 4. Enable the Case Flow in Your Community
 
 1. In your Salesforce community, drag the Lightning Flow component in a Community page and select the `Case_Assist_Recommended_Flow` or the `Case_Assist_Demo_Flow` shipped with this repository.
-2. After selecting the name of the flow, two inputs labeled `caseAssistId` and `engineId` appear. Fill the `caseAssistId` input with your [Case Assist Id](https://docs.coveo.com/en/3328/#retrieving-a-case-assist-id) retrieved from your Case Assist Configuration and fill the `engineId` input with your [Engine Id](https://docs.coveo.com/en/quantic/latest/reference/case-assist-components/case-assist-case-assist-interface/#properties) which is the name you want to give to the engine the Quantic components will register to.
-3. In the published version of your community, users can now fill in the subject and description fields in the first screen and they can proceed to the next screens to see the proposed predictions for classifying their case and get documents suggestions that will potentially resolve their case before it is created.
+2. After selecting the name of the flow, two inputs labeled `caseAssistId` and `engineId` appear.
+   1. Fill the `caseAssistId` input with your [Case Assist Id](https://docs.coveo.com/en/3328/#retrieving-a-case-assist-id) retrieved from your Case Assist Configuration
+   1. Fill the `engineId` input with your [Engine Id](https://docs.coveo.com/en/quantic/latest/reference/case-assist-components/case-assist-case-assist-interface/#properties) which is the name you want to give to the engine the Quantic components will register to.
+   1. Leave the `caseData` blank.
+3. After linking your installed Coveo For Salesforce package with a Coveo organization, make sure to go change the content of the Apex class "CaseAssistController". By default it will try to query a Sample organization. You should replace this method with the commented method just below it to generate a Platform Token and query your selected Coveo Organization.
```

Comment:
I don't think we mentioning anywhere that the C4SF package needs to be installed, I think this should be clearly stated in the setup steps.
Also I think we need to add the C4SF package as a dependency of the `Salesforce Quantic Case Assist Cookbook` package  in the `sfdx-project.json` file like we we currently doing with the Quantic package.

Reply:
Good idea, I would make it a different jira though

Reply:
Jira created